src/main/webapp/jsp/edit-task.jsp

```diff
@@ -0,0 +1,73 @@
+<%@ page import="com.todolist.app.model.Item" %>
+<%@ page import="com.todolist.app.model.Tag" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Edit task</title>
+    <link rel="stylesheet" href="<%= request.getContextPath() %>/css/edit-task.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.0/css/select2.min.css" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.0/js/select2.min.js"></script>
+    <script type="text/javascript" src="<%= request.getContextPath() %>/js/newtask.js"></script>
+</head>
+<body>
+<div class="edittask">
+    <% Item item = (Item) request.getAttribute("item");
+        String uuid = (String) request.getAttribute("uuid");
+        session.setAttribute("uuid", String.valueOf(uuid));%>
```

Comment:
You might add this uuid field as an input in the form, but hidden. After that you will be able to send in to servlet in scope of the request, I guess.